tools/system_libs.py

```diff
@@ -2493,10 +2493,6 @@ def install_system_headers(stamp):
   bin_dest = cache.get_sysroot_dir('bin')
   copytree_exist_ok(bin_src, bin_dest)
 
-  cmake_src = utils.path_from_root('system/lib/cmake')
-  cmake_dest = cache.get_sysroot_dir('lib/cmake')
-  copytree_exist_ok(cmake_src, cmake_dest)
```

Comment:
Why is this no longer needed?

Reply:
This directory contains on the SDL2 and SDL3 cmake files.  They are now installed when the respective port is installed instead.

Reply:
This is in generic code, though, not SDL - does this not affect any other port?